packages/edit-site/src/components/page-pages/index.js

```diff
@@ -142,12 +150,37 @@ export default function PagePages() {
 						</a>
 					);
 				},
+				filters: [ { id: 'author', type: 'enumeration' } ],
+				elements: [
+					{
+						value: '',
+						label: __( 'All' ),
+					},
+					...( authors?.map( ( { id, name } ) => ( {
+						value: id,
+						label: name,
+					} ) ) || [] ),
+				],
 			},
 			{
 				header: __( 'Status' ),
 				id: 'status',
 				accessorFn: ( page ) =>
 					postStatuses[ page.status ] ?? page.status,
+				filters: [ { type: 'enumeration', id: 'status' } ],
+				elements: [
+					{ label: __( 'All' ), value: 'publish,draft' },
```

Comment:
Why are we limiting to "publish" and "draft"? There might be more statuses like "schedule"... Why not use the REST API for this. Also, why "all" has a value and not just "unset" the filter. 

I also believe the "all" could be added automatically and not provided in this elements config.

Reply:
> Why are we limiting to "publish" and "draft"? There might be more statuses like "schedule"... Why not use the REST API for this. Also, why "all" has a value and not just "unset" the filter.

I don't know. This was in the [original query](https://github.com/WordPress/gutenberg/pull/54966/files#diff-44881c176d8762d8e2b3cb7d8a82d1fe82c69002b50817de1663052ec167a618R58). If all statuses are used, we don't need to provide the "unset" value for the endpoint. I'll look into this in a follow-up. 

> I also believe the "all" could be added automatically and not provided in this elements config.

This is addressed at https://github.com/WordPress/gutenberg/pull/55440

Reply:
Listing pages with any status at https://github.com/WordPress/gutenberg/pull/55476